Dockerfile

```
@@ -427,7 +424,7 @@ FROM deps AS build-js
 
 WORKDIR /platform
 
-COPY --from=build-planner /platform/recipe.json recipe.json
+COPY --from=build-planner --parents /platform/recipe.json recipe.json
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)